esmvaltool/diag_scripts/climate_patterns/rename_variables.py

```diff
@@ -0,0 +1,231 @@
+"""Script containing cube re-naming functions for driving scripts.
```

Comment:
I think that the naming conventions here were chosen to reflect custom naming configured for this use-case, so I'm not certain that it would be useful for other metrics. I'll double check this though!